.gitignore

```diff
@@ -40,3 +41,4 @@ test/.coverage-unit/
 
 # Local config file useful for development
 config.local.json
+lisk.json
```

Comment:
Who introduced this file? and why we need to add it to gitignore?

Reply:
It is a file usually created by users who run the application on `pm2`. It shouldn't be a problem.

Reply:
Its not a strict convention to have pm2 file with name of `pm2`, and if some person run lisk with `pm2` why he would ever commit his files? This will limit us if in future we want to introduce a file with this name for any purpose.

So until any file is already used in our workflow, we should not add it to our version control. e.g. I always use `config.local.json` file and I always run `node app -c config.local.json`, but its never intended that I add that file in version control.

Reply:
I just tried `pm2 start app.js --name lisk`, so that file is not created automatically by `pm2.